impeller/typographer/typographer_unittests.cc

```diff
@@ -188,8 +190,20 @@ TEST_P(TypographerTest, GlyphAtlasWithLotsOfdUniqueGlyphSize) {
   ASSERT_NE(atlas, nullptr);
   ASSERT_NE(atlas->GetTexture(), nullptr);
 
-  ASSERT_EQ(atlas->GetTexture()->GetSize().width * 2,
```

Comment:
The aspect ratio of the texture isn't really consequential.  It just matters that there is an atlas that has all the glyphs (and it fits into texture the size the hardware can support).